src/chat.rs

```diff
@@ -574,10 +574,26 @@ impl ChatId {
 
     /// Returns number of messages in a chat.
     pub async fn get_msg_cnt(self, context: &Context) -> Result<usize> {
```

Comment:
i just checked android/ios/desktop - `get_msg_cnt()` seems not to be used anywhere. UI will get just get the whole list, and i do not think this will change somewhen. so.

we could probably also just deprecate `get_msg_cnt` similar to how we [deprecated `get_blocked_cnt`](https://c.delta.chat/deprecated.html) - that way, we would remove  the need to keep complicated conditions in-sync